libsolidity/formal/SymbolicTypes.cpp

```diff
@@ -169,3 +169,32 @@ smt::Expression dev::solidity::maxValue(IntegerType const& _type)
 {
 	return smt::Expression(_type.maxValue());
 }
+
+void dev::solidity::setZeroValue(SymbolicVariable const& _variable, smt::SolverInterface& _interface)
```

Comment:
Hm, can we have an smt namespace? This function names sounds rather generic.

Reply:
We could also just change the function names to something like `setZeroSymbolicValue`